google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -109,14 +112,28 @@ def _send_email(self, summary_data, details_data):
 
             details_data (list): Details of inventory data as a list of dicts.
                 Example: [[{resource_type, count}, {}, {}, ...]
+
+        Raises:
+            InvalidInputError: Indicating that invalid input was encountered.
         """
         LOGGER.debug('Sending inventory summary by email.')
 
         email_summary_config = (
-            self.notifier_config.get('inventory').get('email_summary'))
+            self.notifier_config.get('inventory').get('email_summary')
+        )
```

Comment:
Move this parenthesis back to the previous line.

Reply:
Done